### PR TITLE
Arista EOS show environment power

### PR DESCRIPTION
**Pull request by pauljorgenson**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

Arista EOS `show environment power` command

##### SUMMARY

```

```

### CODE REVIEW

**Comment by diepes** [comment]
Hi @pauljorgenson 
The variable naming does not match with the table headings. The naming and units is not consistent.
e.g.
Table[Output Power]460w  -> power_supply_watts: "460"
Table[Input Current]1.05A -> power_supply_input: "1.05A"

In the one case the Units[w] is dropped and in the other the unit[A] is kept in the output value.
Should we keep the unit in both cases and rename power_supply_watts to power_supply_output ?

Another example is Table[Model] -> power_supply_pid: "PWR-460AC-F" 
would it make sense to rename this to power_supply_model ?

File: ntc_templates/templates/arista_eos_show_environment_power.textfsm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ^%\sThere\sseem\sto\sbe\s${POWER_SUPPLY_STATUS}\sconnected.$$
+  # Physical EOS devices with power supplies
+  ^Power\s+Input\s+Output\s+Output.*$$
+  ^Supply\s+Model\s+Capacity\s+Current\s+Current\s+Power\s+Status\s+Uptime$$
+  ^Supply\s+Model\s+Capacity\s+Current\s+Current\s+Power\s+Status$$
+  ^-.*$$
+  ^${POWER_SUPPLY_LOCATION}\s+${POWER_SUPPLY_PID}\s+${POWER_SUPPLY_WATTS}W\s+${POWER_SUPPLY_INPUT}\s+${POWER_SUPPLY_OUTPUT}\s+${POWER_SUPPLY_WATTS_USING}W\s+${POWER_SUPPLY_STATUS} *${POWER_SUPPLY_UPTIME}\s*$$ -> Record
+  ^${POWER_SUPPLY_LOCATION}\s+${POWER_SUPPLY_PID}\s+${POWER_SUPPLY_WATTS}W\s+${POWER_SUPPLY_INPUT}\s+${POWER_SUPPLY_OUTPUT}\s+${POWER_SUPPLY_WATTS_USING}W\s+${POWER_SUPPLY_STATUS}$$ -> Record
+  ^Total\s+-+\s+${SYSTEM_TOTAL_POWER_WATTS}W\s+-+\s+-+\s+${SYSTEM_TOTAL_POWER_USED_WATTS}W\s+.*$$
+  ^. -> Error
```
Comment (jmcgill298) [inline ntc_templates/templates/arista_eos_show_environment_power.textfsm:29]:
```suggestion
  ^. -> Error

```

File: tests/arista_eos/show_environment_power/arista_eos_show_environment_power1.raw [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Power              Input Output Output
+Supply Model      Capacity Current Current Power Status      Uptime
+------ --------------- -------- ------- ------- ------ ------ ----------------
+1   PWR-1011-AC-RED  1000W  0.70A 11.58A 139.0W Ok   68 days, 0:11:28
+2   PWR-1011-AC-RED  1000W  0.68A 11.31A 134.0W Ok   68 days, 0:11:29
+Total --         2000W   --   -- 273.0W --          --
```
Comment (jmcgill298) [inline tests/arista_eos/show_environment_power/arista_eos_show_environment_power1.raw:6]:
```suggestion
Total --         2000W   --   -- 273.0W --          --

```

File: tests/arista_eos/show_environment_power/arista_eos_show_environment_power2.raw [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Power                                  Input    Output   Output
+Supply  Model                Capacity  Current  Current  Power    Status
+------- -------------------- --------- -------- -------- -------- -------------
+1       PWR-460AC-F               460W    0.00A    0.00A     0.0W Power Loss
+2       PWR-460AC-F               460W    1.05A   10.38A   121.0W Ok
```
Comment (jmcgill298) [inline tests/arista_eos/show_environment_power/arista_eos_show_environment_power2.raw:5]:
```suggestion
2       PWR-460AC-F               460W    1.05A   10.38A   121.0W Ok

```

File: tests/arista_eos/show_environment_power/arista_eos_show_environment_power3.raw [hunk 4]
```diff
@@ -0,0 +1,6 @@
+Power                        Input  Output  Output
+Supply Model       Capacity Current Current  Power Status              Uptime
+------ ----------- -------- ------- ------- ------ ------ -------------------
+1      PWR-500AC-R     500W   0.35A   5.34A  64.8W Ok     1087 days, 21:24:36
+2      PWR-500AC-R     500W   0.38A   5.77A  70.0W Ok       617 days, 8:39:45
+Total  --             1000W      --      -- 134.8W --                      --
```
Comment (jmcgill298) [inline tests/arista_eos/show_environment_power/arista_eos_show_environment_power3.raw:6]:
```suggestion
Total  --             1000W      --      -- 134.8W --                      --

```

File: tests/arista_eos/show_environment_power/arista_eos_show_environment_power4.raw [hunk 5]
```diff
@@ -0,0 +1 @@
+% There seem to be no power supplies connected.
```
Comment (jmcgill298) [inline tests/arista_eos/show_environment_power/arista_eos_show_environment_power4.raw:1]:
```suggestion
% There seem to be no power supplies connected.

```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Comment by ryanmerolle** [comment]
What is required to push this through. I'd be happy to help

**Comment by jmcgill298** [comment]
Fixed requested whitespace updates and rebased to resolve conflicts: https://github.com/networktocode/ntc-templates/pull/1734

**Pull request closed without merging**